libs/template/templates/default-python/template/{{.project_name}}/src/notebook.ipynb.tmpl

```diff
@@ -0,0 +1,65 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "ee353e42-ff58-4955-9608-12865bd0950e",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "# Default notebook\n",
+    "\n",
+    "This default notebook is executed using Databricks Workflows as defined in resources/{{.my_project}}_job.yml."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {
+      "byteLimit": 2048000,
+      "rowLimit": 10000
+     },
+     "inputWidgets": {},
+     "nuid": "6bca260b-13d1-448f-8082-30b60a85c9ae",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "from {{.project_name}} import main\n",
```

Comment:
I doubt this works at the moment, given we don't transparently set $PYTHONPATH or $PWD yet.

Reply:
This appears to work just fine for Jobs + Interactive. We only need PYTHONPATH, not PWD. But perhaps there are older DBRs where PYTHONPATH is not set? @fjakobs could you offer additional insights here?

Reply:
AFAIK python path is only set in Repos but not in workspace.

Reply:
Ok. I don't know what changed but the above works just fine on at least recent DBRs like 12.2+.